lib/registration.pm

```diff
@@ -114,6 +114,13 @@ sub fill_in_registration_data {
                 }
             }
             my @scc_addons = split(/,/, get_var('SCC_ADDONS', ''));
+
+            # Workaround for boo#1056047
+            if (!(check_screen 'scc_module-phub')) {
```

Comment:
Please don't use check_screen with default timeout to not waste any time. `check_screen(…, 0)` should work here

Reply:
Won't it cause an instant fail even if phub would be there?